src/System.CommandLine/ParseResult.cs

```diff
@@ -17,27 +17,29 @@ namespace System.CommandLine
     /// </summary>
     public sealed class ParseResult
     {
-        private readonly IReadOnlyList<ParseError> _errors;
         private readonly CommandResult _rootCommandResult;
         private readonly IReadOnlyList<Token> _unmatchedTokens;
         private CompletionContext? _completionContext;
-        private CliAction? _action;
+        private readonly CliAction? _action;
+        private readonly List<CliAction>? _nonexclusiveActions;
```

Comment:
thank you for making it optional by default and avoiding the allocations for cases where it's not needed 👍